paper-slider.html

```diff
@@ -242,9 +262,19 @@
       '_minChanged(min)',
       '_maxChanged(max)',
       '_valueChanged(value)',
-      '_immediateValueChanged(immediateValue)'
+      '_immediateValueChanged(immediateValue)',
+      '_focusChanged(focused)'
     ],
 
+    hostAttributes: {
+      tabindex: 0
```

Comment:
What is the `role` for this element?

http://www.w3.org/TR/wai-aria/roles#slider